optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -448,7 +436,7 @@ def _normal_cdf(x: np.ndarray, mu: np.ndarray, sigma: np.ndarray) -> np.ndarray:
         denominator = x - mu
         numerator = np.maximum(np.sqrt(2) * sigma, EPS)
         z = denominator / numerator
-        return 0.5 * (1 + special.erf(z))
+        return 0.5 * (1 + np.vectorize(math.erf)(z))
```

Comment:
FYI, I micro-benchmarked this line and got the following results:
```
In [10]: xs = np.linspace(-2, 2, 100000)

In [11]: %timeit np.vectorize(math.erf)(xs)
6.5 ms ± 35.6 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [12]: %timeit special.erf(xs)
631 µs ± 36.7 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```

`np.vectorize(math.erf)` could be 10 times slower than `special.erf`. This could be or not be a problem.